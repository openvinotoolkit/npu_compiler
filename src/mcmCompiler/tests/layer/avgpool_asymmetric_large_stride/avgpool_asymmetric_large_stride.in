version https://git-lfs.github.com/spec/v1
oid sha256:4a24e7bec8bb9054ce517825a71def2316f4cf68352090fad711eb071a38ca04
size 11040
