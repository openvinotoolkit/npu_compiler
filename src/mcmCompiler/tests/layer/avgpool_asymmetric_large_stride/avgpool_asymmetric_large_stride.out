version https://git-lfs.github.com/spec/v1
oid sha256:fed69a3534641940e74b7b95787755fd1ddab430841ca0530507352d2f299b69
size 144
